pkgs/by-name/li/libideviceactivation/package.nix

```diff
@@ -1,55 +1,57 @@
 {
+  lib,
   stdenv,
   fetchFromGitHub,
-  lib,
-  pkg-config,
+
   automake,
   autoreconfHook,
-  libtool,
-  libplist,
+  pkg-config,
+
+  curl,
   libimobiledevice,
+  libplist,
+  libtool,
   libxml2,
-  curl,
   usbmuxd,
 }:
-
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation {
   pname = "libideviceactivation";
-  version = "1.1.1";
+  version = "1.1.1-unstable-2024-05-29";
```

Comment:
We do need an unstable version since the last release is from more than 4 years ago and the developer doesn't want to create new releases/tags.
It's the same reason for why the rest of their packages were made unstable as well.
That and because the developer usually responds to the bug reporters to build the packages from master before reporting and what is in the distro's repos are from the old releases.